changes/4619.fixed

```diff
@@ -0,0 +1 @@
+Fixed broken links in Nautobot README.md.
```

Comment:
Should we add a `documentation` category to towncrier? We seem a bit inconsistent lately about whether docs changes belong under `fixed` or `housekeeping`.

Reply:
I don't think that calling out a typo fix is adding any value to somebody installing and using Nautobot so it should go under housekeeping. If you're adding a new section or overhauling a section of docs then you could continue to use the other categories. Thoughts?

Reply:
I also think a documentation category would be fine too.

Reply:
https://github.com/nautobot/nautobot/issues/4642